### ISSUE TITLE
\e[2m not working properly

### ISSUE THREAD

**Issue by liclac** [asciinema/asciinema-player#52]
(Originally, mistakenly, [posted](https://github.com/asciinema/asciinema.org/issues/236) on the asciinema.org repo.)

The dim/faint color attribute (`\e[2m`) doesn't show up properly in the web player.

Running this script:

```bash
#!/bin/bash

printf "\n"
printf "    _   __                           __   \e[2m______      _       __ \e[0m\n"
printf "   / | / /___  _________ ___  ____ _/ /  \e[2m/ ____/___ _(_)___  / /_\e[0m\n"
printf "  /  |/ / __ \/ ___/ __ ´__ \/ __ ´/ /  \e[2m/ /_  / __ ´/ / __ \/ __/\e[0m\n"
printf " / /|  / /_/ / /  / / / / / / /_/ / /  \e[2m/ __/ / /_/ / / / / / /_  \e[0m\n"
printf "/_/ |_/\____/_/  /_/ /_/ /_/\__,_/_/  \e[2m/_/    \__,_/_/_/ /_/\__/  \e[0m\n"
printf "\n"
```

Produces the expected output in a terminal:

<img width="465" alt="screen shot 2017-02-22 at 19 24 56" src="https://cloud.githubusercontent.com/assets/428026/23267651/ad4948a2-f9eb-11e6-87a7-eccb59cf53ba.png">

<img width="464" alt="screen shot 2017-02-22 at 19 28 06" src="https://cloud.githubusercontent.com/assets/428026/23267656/b01947a8-f9eb-11e6-817c-518c7287aa02.png">

But not in the player:
- black/white: https://asciinema.org/a/4dc8adqp6ydgz4rcph4zvuucb
- colored: https://asciinema.org/a/8muthjs1m0bcbo6f1c4tp8kt1

**Comment by bl-ue** [comment]
@sickill Could I work on this?

**Comment by bl-ue** [comment]
@sickill?

**Comment by bl-ue** [comment]
It also doesn't work for tools that use asciinema like [`svg-term`](https://github.com/marionebl/svg-term-cli).  @sickill is the idea realistic that adding support for `\e[2m` would be as simple as https://github.com/asciinema/vt/commit/fecc4f04c13ceb7984678cc93fa0fde2a192f042?

**Comment by sickill** [comment]
Just released 3.1.0 with the fix.

**Comment by sickill** [comment]
Fixed in 3.1.0.
